src/core/models/gyro_model.py

```diff
@@ -0,0 +1,19 @@
+from core.models.model import SensorModel
+import numpy as np
+
+class GyroModel(SensorModel):
+    """The gyro sensor model. This model applies the gyro bias and noise as specified in parameters.py"""
+
+    def __init__(self, parameters: Parameters) -> None:
+        super().__init__(parameters)
+    
+    def evaluate(self, state: State) -> Dict[str, Any]:
+        return super().evaluate(state)
+    
+    def d_state(self, state: State) -> Dict[str, Any]:
```

Comment:
We'll probably want to change the `SensorModel` base class a little bit, but that's for a future PR